### PR TITLE
Add isUrlFilterCaseSensitive change to What's New

### PR DESCRIPTION
**Pull request by oliverdunk**
Adds the change to the default isUrlFilterCaseSensitive value to What's New.

Fixes https://github.com/GoogleChromeLabs/Extension-Docs/issues/137

### CODE REVIEW

File: site/en/docs/extensions/whatsnew/index.md [hunk 1]
```diff
@@ -15,6 +15,20 @@ tags:
...
+
+Starting in Chrome 118, the [isUrlFilterCaseSensitive](/docs/extensions/reference/declarativeNetRequest/#property-RuleCondition-isUrlFilterCaseSensitive)
+property in the [chrome.declarativeNetRequest](/docs/extensions/reference/declarativeNetRequest/)
+API has been changed to default to false.
+
+This follows [discussions](https://github.com/w3c/webextensions/issues/269) in the Web Extensions
+Community Group and a similar change already implemented by Firefox and Safari.
+
+If you wish to keep the old behavior, you can explicitly set isUrlFilterCaseSensitive to true in
+your declarativeNetRequest rules.
```
Comment (AmySteam) [inline site/en/docs/extensions/whatsnew/index.md:30]:
```suggestion
Starting in Chrome 118, the [`isUrlFilterCaseSensitive`](/docs/extensions/reference/declarativeNetRequest/#property-RuleCondition-isUrlFilterCaseSensitive)
property in the [chrome.declarativeNetRequest](/docs/extensions/reference/declarativeNetRequest/)
API has been changed to default to `false`. If you wish to keep the old behavior, you can explicitly set `isUrlFilterCaseSensitive` to `true` in
your declarativeNetRequest rules.

This follows [discussions](https://github.com/w3c/webextensions/issues/269) in the Web Extensions
Community Group. Firefox and Safari have already implemented a similar change.
```
I feel like this flows better. Right now, it's looking a bit chopped up:

Screenshot 2023-10-10 at 11 46 43

**Review by AmySteam — CHANGES_REQUESTED** [review]
I'm helping out Joe with a couple of reviews. I made a suggestion to improve the flow

**Comment by Mohammed71o** [comment]
Ok,je vais la Petter cette ordi ok, bye

Le mar. 10 oct. 2023, 12 h 49, amysteamdev ***@***.***> a
écrit :

**Review by AmySteam — APPROVED** [review]
LGTM 👍

File: site/en/docs/extensions/whatsnew/index.md [hunk 2]
```diff
@@ -15,6 +15,18 @@ tags:
 
 Check this page often to learn about changes to Chrome extensions, extensions documentation, or related policy or other changes. You'll find other notices posted on the [Extensions Google Group](https://groups.google.com/a/chromium.org/g/chromium-extensions). The [Extensions News](/tags/extensions-news/) tag lists articles about some of the topics listed here. (It even has [an RSS feed](/feeds/extensions-news.xml).) The [Chrome schedule](https://chromiumdash.appspot.com/schedule) lists stable and beta release dates.
 
+### Chrome 118: isUrlFilterCaseSensitive now defaults to false {: #118-url-filter-case-sensitive }
+
+<p class="color-secondary-text type--caption">Posted on <time>October 10, 2023</time></p>
```
Comment (oliverdunk) [inline site/en/docs/extensions/whatsnew/index.md:20]:
```suggestion
<p class="color-secondary-text type--caption">Posted on <time>October 11, 2023</time></p>
```

**Pull request merged**